sway/container.c

```diff
@@ -110,7 +110,7 @@ swayc_t *new_view(swayc_t *sibling, wlc_handle handle) {
 		handle, title, type, sibling, sibling?sibling->type:0);
 	//Setup values
 	view->handle  = handle;
-	view->name    = strdup(title);
+	view->name	= strdup(title);
 	view->visible = true;
```

Comment:
How did I miss this earlier? Style guideline - one space around =, do not align.